src/library/naked.jl

```diff
@@ -0,0 +1,16 @@
+export naked
+
+@api naked => (Naked <: Tile) begin
```

Comment:
I wanted to make it unique for the future as raw/html/etc are very generic I thought there'd be less chance of a conflict.